README.md

```diff
@@ -87,33 +68,29 @@ the build wizard doesn't do.  It adds MIME Types, adds an icon to your
 Application menu, registers icons, adds a /usr/bin/openshot command, and
 copies all the code files to the /site-packages/ folder.
 
-
-
-#################
-    TUTORIAL
-#################
+## Tutorial
 
 Here is a tutorial of the current development features:
 
-	DO YOU HAVE A HELP MANUAL?
+	Do you have a help manual?
```

Comment:
Currently, the heavy indents on the entire Tutorial section are forcing the contents into a \<PRE\> block (which is what Markdown does when any text is indented by 4 or more spaces, or any tabs), causing it to not wrap, force horizontal scrolling, and generally look... poor. 

I'd suggest just pulling out the deep indents and moving everything left so Markdown will auto-format it as a nested list. Or, heck, you could even forget the nesting and use level-4 headers for the "questions" e.g.
```
#### How do I add media to my project? 
  1) Drag and drop videos or music files from gnome into the "Project Files" tree.
  2) Click the File / Import Files... menu
  3) Click the "Import Files" icon on the toolbar (at the top of the screen)
```
Which formats as:

#### How do I add media to my project? 
  1) Drag and drop videos or music files from gnome into the "Project Files" tree.
  2) Click the File / Import Files... menu
  3) Click the "Import Files" icon on the toolbar (at the top of the screen)